tests/experiment/test_experiment.py

```diff
@@ -273,6 +293,12 @@ def test_platform_attributes(self, experiment: Experiment):
         assert not hasattr(experiment, "_plot")
         assert not hasattr(experiment, "_remote_id")
 
+    def test_experiment_settings_attributes_to_str(self, experiment: Experiment):
```

Comment:
Nice new addition! 🔥 

But if this is the reason, for the previous change in the __str__ method, please only do t if its better, and explain it, don't do it to pass codecov :)

Reply:
I am not sure I'm getting your point. I did not changed the serialization of the `ExperimentSettings` class, had to add it to pass codecov. Then I changed the old implementation to the new one for a better readable code thus resulting in a more readable test too. I changed the implementation after your review but I don't get what do you mean by explaining it (you mean on the docstrings?)

Reply:
Nothing I got confused, didn;t realize its a new test! GJ, resolving this one 👍